### DISCUSSION TITLE
NMT Model Checkpointing

### DISCUSSION THREAD

**Discussion by chandrasekaraditya** [NVIDIA-NeMo/NeMo#2504] [Q&A] [answered]
Hello!

Does NeMo provide model check pointing for Machine Translation Models? That way, a saved checkpoint can be restored and be used for training.

Thanks!

**Comment by ericharper** [comment, marked as answer]
Yes, all NeMo models can be saved and restored: https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/core/core.html#save-and-restore
